packages/core/cli/src/commands/init.ts

```diff
@@ -22,16 +29,66 @@ export default async (args) => {
     projectName = typedProjectName;
   }
 
-  const integrations = getIntegrations();
-  const { chosenIntegration } = await inquirer
-    .prompt([
-      {
-        type: 'list',
-        name: 'chosenIntegration',
-        message: 'Choose integration',
-        choices: integrations
+  const { chosenIntegration } = await inquirer.prompt([
+    {
+      type: 'list',
+      name: 'chosenIntegration',
+      message: 'Choose integration',
+      choices: [...integrationsNames, CUSTOM_TEMPLATE]
+    }
+  ]);
+
+  if (chosenIntegration !== CUSTOM_TEMPLATE) {
```

Comment:
in this file you have the perfect scenario for a sort of strategy pattern:  eg. you can create a context function that just receives input parameters and base on these parameters (as you have if's here) you can run a specific strategy. Each strategy function is a different creation of project, for example:


```js
const customTemplateStrategy = () => { ... }
const integrationStrategy = () => { ... }
const defaultStrategy = () = { ... }

const selectStrategy = (params) => {
 if (is custom Template) {
   return customTemplateStrategy 
 }

 if ( is integration ) {
  return integrationStrategy 
 }

 return defaultStrategy 
}


// init.ts
const strategy = selectStrategy(...)
strategy(...)
```